js/src/edit-block/hooks/useField.js

```diff
@@ -223,6 +224,26 @@ const useField = () => {
 		editPost( { content: JSON.stringify( fullBlock ) } );
 	}, [ blockNameWithNameSpace, editPost, fullBlock ] );
 
+	/**
+	 * Duplicates this field.
+	 */
+	const duplicateField = useCallback( ( fieldName ) => {
```

Comment:
It looks like when this is called, it doesn't pass an argument.

Reply:
Or it would just be passed the `event` object.

Reply:
Hmm, I'm not 100% sure, but my thinking was that [this line](https://github.com/studiopress/genesis-custom-blocks/pull/53/files#diff-84d96f93e4494795fbab4f77ff96fe295b0e062e868b2f5b7498ac724b1af5aeR109) would be passing the `selectedField` just like in the `deleteField`.

Reply:
OK, sounds good

Reply:
You're right, I missed that